slips_files/common/parsers/config_parser.py

```diff
@@ -82,7 +85,7 @@ def web_interface_port(self) -> int:
 
     def get_entropy_threshold(self):
         """
-        gets the shannon entropy used in detecting C&C over DNS TXT records from slips.conf
+        gets the shannon entropy used in detecting C&C over DNS TXT records from slips.conf/slips.yaml
```

Comment:
all comments saying slips.conf/slips.yaml should say slips.yaml only, since slips.conf will be deleted for good. and also you should delete the slips.conf from git in your PR, we will no longer need it right?